examples/step-67/doc/results.dox

```diff
@@ -0,0 +1,390 @@
+    <td align="center">2.789e-11</td>
+    <td>3.94</td>
+  </tr>
+  <tr>
+    <td align="right">262,144</td>
+    <td align="center">9,437,184</td>
+    <td align="center">2.038e-06</td>
+    <td>2.86</td>
+    <td align="center">16,777,216</td>
+    <td align="center">3.609e-09</td>
+    <td>4.01</td>
+    <td align="center">37,748,736</td>
+    <td align="center">5.73e-11</td>
+    <td>-1.0</td>
+  </tr>
+</table>
+
+The tables show that we get optimal $\mathcal O\left(h^{p+1}\right)$
+convergence rates for both numerical fluxes. The errors are slighly smaller
+for the Lax--Friedrichs flux for $p=2$, but the picture is reversed for
+$p=3$. For $p=5$, we reach the roundoff accuracy of $10^{-11}$ with both
+fluxes on the finest grids. Also note that the errors are absolute, and the
+domain length is $10^2$, so relative errors are below $10^{-12}$. The HLL flux
+is somewhat better for the highest degree, which is due to a slight inaccuracy
+of the Lax--Friedrichs flux: For very accurate results, setting a Dirichlet
+condition on the solution that leaves the domain results in a small artificial
+reflection, which is accentuated for the Lax--Friedrichs flux. Apart from
+that, we see that the influence of the numerical flux is minor. This also
+holds for other scenarios. For example taking the setup of step-33, we get
+oscillations (which in turn make density negative and make the solution
+explode) with both fluxes once the high-mass part comes near the
+boundary. That case would clearly need some form of limiting.
+
+<h3>Results for flow in channel around cylinder in 2D</h3>
+
+<h3>Results for flow in channel around cylinder in 3D</h3>
+
+<h3>Possibilities for extensions</h3>
+
+<h4>Equipping the code for supersonic calculations</h3>
+
+As mentioned in the introduction, the solution to the Euler equations develops
+shocks as the Mach number increases, which require additional mechanisms to
+stabilize the scheme, e.g. in the form of limiters. The main challenge in that
+case would be to load-balance the computations, as the additional computations
+involved for limiting the oscillations in troubled cells would make them
+heavier than the plain DG cells without limiting. Furthermore, additional
+numerical fluxes that better cope with the discontinuities would also be an
+option.
+
+One ingredient also necessary for supersonic flows are appropriate boundary
+conditions. As opposed to the subsonic outflow boundaries discussed in the
+introduction and implemented in the program, all characteristics are outgoing
+for supersonic outflow boundaries, so we do not want to prescribe any external
+data,
+@f[
+\mathbf{w}^+ = \mathbf{w}^- = \begin{pmatrix} \rho^-\\
+(\rho \mathbf u)^- \\ E^-\end{pmatrix} \quad
+ \text{(Neumann)}.
+@f]
+
+In the code, we would simply add the additional statement
+@code
+            else if (supersonic_outflow_boundaries.find(boundary_id) !=
+                     supersonic_outflow_boundaries.end())
```

Comment:
isn't it the case that whether or not an outflow boundary is supersonic depends on the solution, not just where you prescribe boundary indicators?